site/gatsby-site/src/components/forms/SubmitForm.js

```diff
@@ -101,27 +89,43 @@ const SubmitForm = () => {
   `);
 
   useEffect(() => {
-    if (!loading) {
-      const submission = { ...query, cloudinary_id: '' };
+    let submission = { ...query, cloudinary_id: '' };
```

Comment:
You can keep the `const` declaration here.

Reply:
I can't because I'm actually assigning the value on line 109